.gitignore

```diff
@@ -1,4 +1,5 @@
 .vagrant
+.idea
```

Comment:
this is specific to your development environment, correct?  You should probably add this to a global `~/.gitignore` instead of every project you open up with your IDE

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/